Tasks/CopyFilesToAzureBlob/task.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+    "id": "EB72CB01-A7E5-427B-A8A1-1B31CCAC8A43",
+    "name": "CopyFilesToAzureBlob",
+    "friendlyName": "Copy Files To Azure Blob",
+    "description": "Copy Files To Azure Blob",
+    "category": "Utility",
+    "visibility": [
+                 "Preview",
+                 "Build",
+                 "Release"
+                  ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0,
+        "IsTest": false
+    },
+    "demands" : [
+                   "azureps"
+                ],
+
+    "inputs": [
+        { 
+            "name": "ConnectedServiceName", 
+            "type": "azureConnection", 
+            "label": "Azure Subscription", 
+            "defaultValue": "", 
+            "required": true,
+            "helpMarkDown": "Azure subscription to configure before running PowerShell"
+        },
+        {
+            "name": "SourcePath",
+            "type": "string",
+            "label": "Source",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Location of the source folder or file. Variables like $(Build.Repository.LocalPath) can be used for binaries in the Automation Agent. For example, $(Build.Repository.LocalPath)\\BudgetIT\\Web." 
```

Comment:
Thanks Omesh/Chris. I have removed the $ variables that are specific to Build from Help strings.